development/image-syncer/external-images.yaml

```
@@ -33,8 +33,8 @@ images:
 - source: "oryd/oathkeeper-maester:v0.1.5"
   amd64Only: true
 - source: "oryd/oathkeeper:v0.38.25-beta.1"
-- source: "postgres@sha256:4a6be41f4176af134fe8a02b9f07928fab22fb1fb2777811360154912cb682cd"
-  tag: "11.18-alpine3.17-v1"
+- source: "postgres@sha256:d83c4c820461e77dca5ac71a54874342be5d225b146b5acdb69c3d107b2c2b47"
```

`docker run mplatform/manifest-tool inspect postgres:11.19-alpine3.17`
We want to use multi platform SHA where possible in case we start supporting arm64 in the future
```suggestion
- source: "postgres@sha256:ee30e7ceb3588f5426fe80db5a1a4b4253c258cb9a9634680552fee5e01efb51c2b47"
```

---

All right, got it.